HtmlBuilderCore/Elements/HtmlList.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HtmlBuilderCore.Elements
+{
+    public class HtmlList : IHtmlElement
+    {
+        public string ListType { get; set; }
```

Comment:
Make an enum : `public enum ListType { Ordered, Unordered};`  Put it in it's own file and use it here instead of string.